fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/BulkExportJobExecutionResponse.java

```diff
@@ -160,84 +160,6 @@ public void setExitStatus(String exitStatus) {
         this.exitStatus = exitStatus;
     }
 
-    public static class JobParameter {
```

Comment:
what replaced this ?

Reply:
added a comment related to this change in JobParameter.java

Reply:
yeah, in a separated fine now, and the job parameters are commonly used ...